ting_material_details.field.inc

```diff
@@ -457,6 +463,36 @@ function ting_material_details_field_formatter_view($entity_type, $entity, $fiel
             '#markup' => $entity->serieDescription,
           );
           break;
+
+        case 'ting_details_is_part_of_search':
+          $type = $uri = NULL;
+          foreach ($entity->getRelations() as $relation) {
+            $type = explode(':', $relation->getType());
+            if ( isset($type[1]) && $type[1] == 'isPartOfManifestation' ) {
+              $uri  = $relation->getURI();
+            }
```

Comment:
that's actually an error in ting_relation.field.inc: $relation->type return "dbcbib:isPartOfManifestation". As do $relation->getType()